backend/tournesol/views/polls.py

```diff
@@ -55,6 +74,12 @@ class PollRecommendationsBaseAPIView(PollScopedViewMixin, ListAPIView):
     It doesn't define any serializer, queryset nor permission.
     """
 
+    def _metadata_from_filter(self, filtr: str):
+        """
+        _metadata_from_filter("metadata[language]") -> "language"
+        """
+        return filtr.split("[")[1][:-1]
```

Comment:
No, I didn't think about it.

I used `metadata[uploader]` because in the poll API we are already using this syntax to filter by criteria ( `weights[better_habits]`), and I didn't want to add a new behaviour here.

Reply:
Ok! Thanks for the detailed explanation


What do you think of making effort such that query parameters in the frontend urls fit more closely to query parameters of the API ?

Reply:
I would say it's a good idea.

We should check if the syntax `weights[better_habits]` is the default behaviour of DRF or if this syntax is enforced by the OpenAPI specifications. If yes we might want to update the front end instead of the back end.